### DISCUSSION TITLE
manual download of packed chrome extension

### DISCUSSION THREAD

**Discussion by AzothZephyr** [phantom/docs#87] [Help & Support] [answered]
### Summary

I use [unchrome](https://github.com/ungoogled-software/ungoogled-chromium), which removes support for installing extensions using the chrome web store. I typically build open source extensions manually or download the extension from the repositories releases page then install manually.

Is there a manner at which users can download a zipped / packed version of the phantom chrome extension outside of the google web store? 

### Steps to Reproduce

1. Install 'ungoogled' chromium https://github.com/ungoogled-software/ungoogled-chromium
2. Try to install phantom 

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hey @AzothZephyr,

Thank you for writing in. Currently this is not an option we offer. I would recommend posting this feature request on our Canny board here: https://phantom.canny.io
